tests/flow/test_comprehension_functions.py

```
@@ -521,3 +521,76 @@ def test21_aggregation_in_pattern_comprehension(self):
         except redis.exceptions.ResponseError as e:
             self.env.assertIn("Invalid use of aggregating function", str(e))
 
+    def test22_comprehension_predicate_after_with(self):
+        # Validate that aliases referenced within comprehension predicate
+        # (any/all/single/none) private-data are visible to the planner
+        g = self.db.select_graph("comprehension_after_with")
+
+        # delete graph in case it exists
+        try:
+            g.delete()
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)